packaging/Makefile_distro_vars.mk

```diff
@@ -69,4 +69,19 @@ ifeq ($(ID_LIKE),suse)
 define install_repo
 	zypper --non-interactive ar $(1)
 endef
-endif
\ No newline at end of file
+endif
+ifeq ($(ID_LIKE),debian)
+ifndef LANG
+export LANG = C.UTF-8
+endif
+ifndef LC_ALL
+export LC_ALL = C.UTF-8
+endif
+else
+ifndef LANG
+export LANG = C.utf8
+endif
+ifndef LC_ALL
+export LC_ALL = C.utf8
+endif
+endif
```

Comment:
Did this work?  If so, it should go to [packaging/](https://github.com/daos-stack/packaging/)